provider/storage_benchmark.py

```diff
@@ -342,11 +342,12 @@ def __init__(self, params, session):
         self.fio_path = '%s/bin/fio' % self.fio_inst
         scp_benckmark = attrgetter('scp_benckmark')
         unpack_file = attrgetter('unpack_file')
+        install_timeout = params.get('fio_install_timeout', 300)
```

Comment:
@ngu-niny Can use `params.get_numeric()` to return a numeric, so you don't need to convert it to an integer next.

Reply:
@PaulYuuu Updated and retest PASS.